src/sc_crawler/vendors/alicloud.py

```
@@ -0,0 +1,849 @@
+                "api_reference": region.get("RegionId"),
+                "display_name": f"{location['city']} ({location['country_id']})",
+                "aliases": location.get("alias", []),
+                "country_id": location.get("country_id"),
+                "state": None,  # not available
+                "city": location.get("city"),
+                "address_line": None,  # not available
+                "zip_code": None,  # not available
+                "lon": location.get("lon"),
+                "lat": location.get("lat"),
+                "founding_year": location.get("founding_year"),
+                # Clean electricity accounted for 56.0% of the total electricity consumption at Alibaba Cloud’s self-built data centers
```

Ruff (0.14.10)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)